install_config/master_node_configuration.adoc

```diff
@@ -1761,39 +1763,63 @@ $ openshift start node --config=/openshift.local.config/node-<node_hostname>/nod
 |8|Body-level API debugging information
 |===
 
-You can control which INFO messages are logged by setting the loglevel option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files], 
-the *_/etc/origin/master/master-config.yaml_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
+You can xref:master-node-config-logging-levels[change the log levels independently for masters or nodes] as needed.
 
-[NOTE]
-====
-Messages with FATAL, ERROR, WARNING and some INFO severities appear in the logs regardless of the log configuration.
-====
+*View node logs*
 
-You can view logs for the master or the node system using the following command:
+To view logs for the node system, use the following command:
 
 ----
 # journalctl -r -u <journal_name>
 ----
 
 Use the `-r` option to show the newest entries first.
 
+*View master logs*
+
+To view logs for the master components, use the following command:
+
+----
+# /usr/local/bin/master-logs <component> <container name> <1>
+----
+
+<1> Specify the component, `container` `api`, or `etcd` and the name of the container. 
+
 For example:
 
 ----
-# master-logs controllers controllers
-# master-logs api api
-# journalctl -r -u atomic-openshift-node.service
+# /usr/local/bin/master-logs controllers controllers
+# /usr/local/bin/master-logs api api
+# /usr/local/bin/master-logs etcd etcd
+----
+
+*Redirect master log to a file*
+
+To redirect the output of master log into a file, use the following command:
+
+----
+master-logs api api 2> file
 ----
 
+[[master-node-config-logging-levels]]
+=== Configuring Logging Levels
+
+You can control which INFO messages are logged by setting the `DEBUG_LOGLEVEL` option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files] or the */etc/origin/master/master.env* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
```

Comment:
> option in the ~in~

Remove the 2nd ~in~

Reply:
s/Collecting all messages should only be used in debug situations./Only collect all messages when you need to debug your cluster.